ec2utils/ec2utilsbase/lib/ec2utils/ec2utilsutils.py

```diff
@@ -76,10 +75,10 @@ def find_images_by_name(images, image_name):
     """Return a list of images that match the given name."""
     matching_images = []
     for image in images:
-        if not image.name:
+        if not image['Name']:
```

Comment:
How did you end up with dicts ? You should have a collection of [`Image`](https://boto3.readthedocs.org/en/latest/reference/services/ec2.html#image) from [`EC2.ServiceResource#images`](https://boto3.readthedocs.org/en/latest/reference/services/ec2.html#EC2.ServiceResource.images)